src/plugins/intel_npu/src/plugin/npuw/compiled_model.hpp

```diff
@@ -154,6 +163,12 @@ class CompiledModel : public ov::npuw::ICompiledModel {
 
         // Metrics
         execution_stats stat;
+
+        void serialize(std::ostream& stream, const std::size_t& idx, const std::string& device) const;
+        static CompiledModelDesc deserialize(std::istream& stream,
+                                             const std::size_t idx,
+                                             const std::shared_ptr<const ov::IPlugin>& plugin,
+                                             const ov::AnyMap& properties);
```

Comment:
There's way too much context passed in there, you certainly don't need that much.

Reply:
Done